src/schema/v2/artist/__tests__/helpers.test.ts

```diff
@@ -0,0 +1,162 @@
+import { ARTIST_INSIGHT_MAPPING, getArtistInsights } from "../helpers"
+
+describe("getArtistInsights", () => {
+  it("returns artist objects with no other information for each insight kind", () => {
+    const artist = {}
+    const insights = getArtistInsights(artist)
+    const mappingLength = Object.entries(ARTIST_INSIGHT_MAPPING).length
+    expect(insights.length).toEqual(mappingLength)
+    insights.forEach((insight) => {
+      expect(insight).toMatchObject({ artist })
+    })
+  })
+
+  describe("pipe delimited insight fields", () => {
+    const value =
+      "Art Institute of Chicago |Brooklyn Museum |       Hamburger Bahnhof"
+
+    const fields = [
+      {
+        key: "solo_show_institutions",
+        kind: "SOLO_SHOW",
+        value,
+      },
+      {
+        key: "group_show_institutions",
+        kind: "GROUP_SHOW",
+        value,
+      },
+      {
+        key: "review_sources",
+        kind: "REVIEWED",
+        value,
+      },
+      {
+        key: "biennials",
+        kind: "BIENNIAL",
+        value,
+      },
+    ]
+
+    fields.forEach((field) => {
+      it(`returns an array of ${field.key} entities split by pipe`, () => {
```

Comment:
Ooh, i've never considered putting an `it` within a loop, but why not?

Reply:
`it.each` would do what you want here w/i the rtl paradigm

Reply:
I tried this but wasn't able to get it to work the way I expected. I followed the docs (I think!) but was not able to get the test titles to do string interp.

```
    pipe delimited insight fields
      ✓ returns an array of $variable.key entities split by pipe
      ✓ returns an array of $variable.key entities split by pipe
      ✓ returns an array of $variable.key entities split by pipe (1ms)
      ✓ returns an array of $variable.key entities split by pipe
```

For whatever reason that `$variable.key` construct from the docs doesn't expand?

https://jestjs.io/docs/api#testeachtablename-fn-timeout